### PR TITLE
Logout functionality from navbar

### PR DESCRIPTION
**Pull request by ahmedhamodi**
I noticed when screen sharing with Amolik that his default flow was to click this button on the navbar that did nothing, so wanted to implement some logic so that it actually works as expected!

Loom video:
https://www.loom.com/share/75dbe1c66ca945898074dc80368abc03

### CODE REVIEW

File: src/Components/Admin/navbar.js [hunk 1]
```diff
@@ -49,39 +54,56 @@ export default function Navbar({ home }) {
...
+                                    onClick={async () => {
+                                        handleClose();
+                                        history.push("/login");
+                                        auth.signOut();
+                                        await httpGet(
+                                            process.env
+                                                .REACT_APP_ADMIN_BASE_ENDPOINT +
+                                                "admin_logout"
+                                        );
+                                    }}
```
Comment (aaronabraham311) [inline src/Components/Admin/navbar.js:100]: I would prefer to have this pulled into another function than make it inline with `onClick`

File: src/Components/Admin/navbar.js [hunk 2]
```diff
@@ -49,39 +54,56 @@ export default function Navbar({ home }) {
...
-                            className={classes.menu}
-                        >
-                            <MenuItem onClick={handleClose}>
-                                Documentation
-                            </MenuItem>
-                            <MenuItem onClick={handleClose}>Settings</MenuItem>
-                            <MenuItem onClick={handleClose}>Logout</MenuItem>
-                        </Menu>
-                    </div>
+                    {user && (
```
Comment (aaronabraham311) [inline src/Components/Admin/navbar.js:57]: I assume this would prevent the menu from even showing up if you're logged out?

**Review by aaronabraham311 — COMMENTED** [review]
Couple questions, overall looks good

File: src/Components/Admin/navbar.js (hunk 2 above)
Comment (ahmedhamodi) [reply src/Components/Admin/navbar.js:57]: The menu bar loads but the right most icon doesn't load so you cannot instigate menu.

Reason for this -- if we have a name or logo present in the navbar, we would probably still want that to load on login page. But no functionality (home, that menu dropdown) should load.

Comment (aaronabraham311) [reply src/Components/Admin/navbar.js:57]: @andstun Perfect example of conditional rendering! You will need it in an upcoming ticket

**Review by aaronabraham311 — APPROVED** [review]

**Pull request merged**